concepticondata/conceptlists/Gabelentz-1892-1074.tsv

```diff
@@ -0,0 +1,1075 @@
+Gabelentz-1892-1074-567	567	Salbe	ointment	III. Substantiva	Y. Nahrung	1926	OINTMENT
+Gabelentz-1892-1074-568	568	Arznei	medicine	III. Substantiva	Y. Nahrung	1372	MEDICINE
+Gabelentz-1892-1074-569	569	Tabak	tobacco	III. Substantiva	Y. Nahrung	974	TOBACCO
+Gabelentz-1892-1074-570	570	Gemüse	vegetables	III. Substantiva	Y. Nahrung	924	VEGETABLES
+Gabelentz-1892-1074-571	571	Ding	thing	III. Substantiva	Z. Allgemeines	738	THING
+Gabelentz-1892-1074-572	572	Stück	piece	III. Substantiva	Z. Allgemeines	594	PIECE
+Gabelentz-1892-1074-573	573	Theil	piece, part, fraction, portion	III. Substantiva	Z. Allgemeines	779	PART
+Gabelentz-1892-1074-574	574	Geschäft	shop	III. Substantiva	Z. Allgemeines	746	SHOP
+Gabelentz-1892-1074-575	575	Markt	market	III. Substantiva	Z. Allgemeines	633	MARKET
+Gabelentz-1892-1074-576	576	Maass	(old German volume measurement unit)	III. Substantiva	Z. Allgemeines		
+Gabelentz-1892-1074-577	577	Klafter	fathom (old unit of measurement)	III. Substantiva	Z. Allgemeines	81	FATHOM
+Gabelentz-1892-1074-578	578	Kreuz	cross	III. Substantiva	Z. Allgemeines	1367	CROSS
+Gabelentz-1892-1074-579	579	Gross	big, tall	IV. Adjectiva	A. Gross u.s.w.	1202	BIG
+Gabelentz-1892-1074-580	580	Hoch	high	IV. Adjectiva	A. Gross u.s.w.	711	TALL
+Gabelentz-1892-1074-581	581	Lang	long	IV. Adjectiva	A. Gross u.s.w.	1203	LONG
+Gabelentz-1892-1074-582	582	Stark	strong	IV. Adjectiva	A. Gross u.s.w.	785	STRONG
+Gabelentz-1892-1074-583	583	Mächtig	powerful, mighty	IV. Adjectiva	A. Gross u.s.w.	3188	MIGHTY OR POWERFUL OR STRONG
+Gabelentz-1892-1074-584	584	Fett	fat	IV. Adjectiva	A. Gross u.s.w.	323	FAT (ORGANIC SUBSTANCE)
+Gabelentz-1892-1074-585	585	Breit	broad	IV. Adjectiva	A. Gross u.s.w.	2531	BROAD
+Gabelentz-1892-1074-586	586	Weit, geräumig	wide	IV. Adjectiva	A. Gross u.s.w.	1243	WIDE
+Gabelentz-1892-1074-587	587	Tief	deep	IV. Adjectiva	A. Gross u.s.w.	1593	DEEP
+Gabelentz-1892-1074-588	588	Alt	old	IV. Adjectiva	A. Gross u.s.w.	1229	OLD
+Gabelentz-1892-1074-589	589	Schwer	heavy	IV. Adjectiva	A. Gross u.s.w.	1210	HEAVY
+Gabelentz-1892-1074-590	590	Klein	small	IV. Adjectiva	B. Klein u.s.w.	1246	SMALL
+Gabelentz-1892-1074-591	591	Schmal	narrow	IV. Adjectiva	B. Klein u.s.w.	1267	NARROW
+Gabelentz-1892-1074-592	592	Niedrig	low	IV. Adjectiva	B. Klein u.s.w.	3348	LOW (POSITION OR HEIGHT)
+Gabelentz-1892-1074-593	593	Mager	meagre	IV. Adjectiva	B. Klein u.s.w.	1400	THIN (SLIM)
+Gabelentz-1892-1074-594	594	Kurz	short	IV. Adjectiva	B. Klein u.s.w.	1645	SHORT
+Gabelentz-1892-1074-595	595	Dünn	thin	IV. Adjectiva	B. Klein u.s.w.	2308	THIN
+Gabelentz-1892-1074-596	596	Schwach	weak	IV. Adjectiva	B. Klein u.s.w.	1601	WEAK
+Gabelentz-1892-1074-597	597	Leicht	light	IV. Adjectiva	B. Klein u.s.w.	1052	LIGHT (WEIGHT)
+Gabelentz-1892-1074-598	598	Eng	tight	IV. Adjectiva	B. Klein u.s.w.	3053	TIGHT
+Gabelentz-1892-1074-599	599	Flach	flat	IV. Adjectiva	B. Klein u.s.w.	1633	FLAT
+Gabelentz-1892-1074-600	600	seicht	shallow	IV. Adjectiva	B. Klein u.s.w.	193	SHALLOW
```

Comment:
It isn't. A handful of words were subsumed under one number in the original list, probably because they are similar in meaning, and the second one of those words was then not capitalized (see also e.g. 616 & 617, Locker & lose).